rust/Cargo.toml

```diff
@@ -24,6 +24,12 @@ itertools = "0.11.0"
 lazy_static = "1.4.0"
 thiserror = "1.0.41"
 
+# Dev tools
+file-header = { version = "0.1.1", optional = true }
+globset = { version = "0.4.13", optional = true }
+# anyhow
+# clap
```

Comment:
leftovers?

Reply:
I wanted to signify that these dependencies are used by Dev tools but declared elsewhere. Though I guess its not important. If the dependencies are deleted, the compiler will complain.

Reply:
Removed.